src/accel/cuda/GPUSolver.cu

```diff
@@ -1423,13 +1489,25 @@ void GPUSolver::storeFSRFluxes() {
 }
 
 void GPUSolver::computeStabilizingFlux() {
-  // TODO
-  log_printf(ERROR, "computeStabilizingFlux() is not yet implemented on GPU.");
+  if (!_stabilize_transport) return;
+
+  if (_stabilization_type == GLOBAL) {
+    FP_PRECISION* scalar_flux = thrust::raw_pointer_cast(&_scalar_flux[0]);
+    FP_PRECISION* stabilizing_flux = thrust::raw_pointer_cast(&_stabilizing_flux[0]);
+    computeStabilizingFluxOnDevice<<<_B, _T>>>(scalar_flux, stabilizing_flux);
+  }
+  else
+    log_printf(ERROR, "Only global stabilization works on GPUSolver now.");
```

Comment:
The default stabilization option (DIAGONAL) makes the GPUSolver run into this error. 
Let's change the default stabilization type to be GLOBAL for the GPUSolver

Reply:
OK, I just put that in the GPUSolver constructor then.